Examples/KerrBH/params.txt

```diff
@@ -144,4 +144,37 @@ sigma = 0.3
 # min_lapse = 1.e-4
 
 #################################################
+# Apparent Horizon Finder parameters
+
+AH_activate = 1 
```

Comment:
Can we call this AH_activated? As it is a bool it should ask a question - AH_activated? True!

Reply:
I  disagree with this. When Tiago renamed the parameters to be prefixed by `AH_` this was just meant as naming scope (like AMReX has `<scope>.<parameter>`). For consistency with `activate_extraction`, I think we should revert back to `AH_activate`.

Reply:
Ok boss!